### PR TITLE
MM-14848 Update the README and values.yaml with clearer instructions

### PR DESCRIPTION
**Pull request by jwilander**
Made some major updates to the README to make the instructions and prerequisites clearer. I ran through these steps myself with a fresh Kubernetes cluster to help me make the improvements.

Goal is to ask some help from the SAs to try these new steps out so they can better help customers who want to run in Kubernetes.

### CODE REVIEW

**Comment by jwilander** [comment]
The chart-lint is failing because I didn't bump the version. As this is just comment/README changes I don't think it's necessary to do that.

**Comment by jasonblais** [comment]
Scanning through, the doc updates look great, really nice work here 👍

**Comment by jwilander** [comment]
1. Sooner than later, but no rush
2. Nope, shouldn't affect that 👍

**Comment by jwilander** [comment]
All good edits, thanks @jasonblais !

File: README.md [hunk 1]
```diff
@@ -1,110 +1,197 @@
...
 
-To use the Mattermost Enterprise Helm Chart, you will first need a running Kubernetes cluster or Minikube.
+# 1. Prerequisites
 
-If you want to expose the application outside of your network, configure ingress in the Kubernetes cluster. Internally, Mattermost uses the [nginx-ingress](https://github.com/kubernetes/ingress-nginx). To install this in your Kubernetes cluster, follow [this documentation](https://kubernetes.github.io/ingress-nginx/deploy/). You may also use the [helm charts](https://github.com/helm/charts/tree/master/stable/nginx-ingress) directly.
+## 1.1 Kubernetes Cluster
 
-Moreover, if you want to automatically retrieve SSL certificates, use [cert-manager](https://github.com/jetstack/cert-manager). You also can use [kube-lego](https://github.com/jetstack/kube-lego), but it is deprecated. You can choose the one that you are most comfortable with. For cert-manager, you can follow [this documentation](https://cert-manager.readthedocs.io/en/latest/) and here is its [helm charts](https://github.com/helm/charts/tree/master/stable/cert-manager).
+You need a running Kubernetes cluster. If you do not have one, find options and installation instructions here:
```
Comment (jasonblais) [inline README.md]:
```suggestion
You need a running Kubernetes cluster v1.8+. If you do not have one, find options and installation instructions here:
```

File: README.md [hunk 2]
```diff
@@ -1,110 +1,197 @@
...
+See: https://docs.helm.sh/using_helm/#quickstart
 
-Depending on the DNS service you're using, the exact steps will differ but generally to point your domain name at your release you can do the following:
+Once helm is installed and initialized, run the following:
 
-Note that your helm release must already be installed and running.
+```bash
+helm repo add mattermost https://releases.mattermost.com/helm
+helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com/
+```
```
Comment (jasonblais) [inline README.md]:
```suggestion
```

We recommend installing Helm v2.13.1 or later.
```

Comment (jwilander) [reply README.md]: I don't think that's the correct change is it?

Comment (jwilander) [reply README.md]: Oh GitHub is screwing it up because of the extra ticks. I see the change and will make it manually

**Comment by jwilander** [comment]
@cpanato PTAL

**Pull request merged**